aws/terraform/modules/common/main.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
+  required_version = ">= 0.12"
```

Comment:
I see the required version as `>=0.13` elsewhere in the PR. Is that intended?

Reply:
my original thinking was this module doesn't have any .13 requirements but we can change it to be consistent